config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -292,7 +292,7 @@ periodics:
     testgrid-tab-name: containerd-node-e2e-1.5
 - name: ci-containerd-node-e2e-1-6
   cluster: k8s-infra-prow-build
-  interval: 1h
+  interval: 24h
```

Comment:
For the 1.6 jobs, should we keep the interval still at 1hr instead of 24hr because its an active branch?

Reply:
since the jobs were failing without anyone noticing or fixing them, running them hourly didn't seem that important